client/src/GamemasterInGame/PlayerListComponents/PlayerMessage/MessageCreators/AlignmentChangeMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import { Alignment, PlayerMessageCreatorMap } from "@hidden-identity/shared";
+import { AlignmentSelect } from "../../Selectors";
+import { useState } from "react";
+import { oppositeAlignment } from "@hidden-identity/shared";
+import { Flex, Heading } from "@radix-ui/themes";
+import { useGetPlayerAlignment } from "../../../../store/useStore";
+import { SubmitMessage } from "../messageShared/SubmitMessage";
+
+export interface AlignmentChangeMessageProps {
+  message: PlayerMessageCreatorMap["alignment-change"];
+  player: string;
+}
+
+export function AlignmentChangeMessage({
+  player,
+}: AlignmentChangeMessageProps) {
+  const getPlayerAlignment = useGetPlayerAlignment();
+  const [alignment, setAlignment] = useState<Alignment>(
+    oppositeAlignment(getPlayerAlignment(player)),
```

Comment:
It feels a little backwards to me to have messages representing future intended game state.  I think if we're displaying an alignment change message I'd prefer if the change happened first and the message reflected that change post-facto.

I think that means we have to decide if actions are executed before messages are shown or vise versa as a whole tho, which is a problem I need to consider carefully.

Reply:
My bad this is a factory not a message itself this is fine.

Reply:
I'm thinking we should create a prefix for these components so they better represent what they are.  "CreateAlignmentChangeMessage" I think would be nice, even while these are contained within a "MessageCreators" folder.